### PR TITLE
Filters for contents list and users list

### PR DESCRIPTION
**Pull request by deanmarcussen**
By and large replaces the existing contents and user filters list for a parser which supports a simple query string language.

Demo'd here https://www.youtube.com/watch?v=1H7XJqP0EyM

Syntax docs etc to follow

### CODE REVIEW

**Comment by agriffard** [comment]
Filtering on Type = Menu (that does not have any results) and then on Culture = en returns an error:

`SqliteException: SQLite Error 1: 'no such column: LocalizedContentItemIndex_a1.ModifiedUtc'.`

**Comment by deanmarcussen** [comment]
Yes, thanks @agriffard some of the defaults are still not working as they need to.

File: src/OrchardCore.Build/Dependencies.props [hunk 1]
```diff
@@ -45,11 +45,11 @@
...
     <PackageManagement Include="xunit.runner.visualstudio" Version="2.4.3" />
-    <PackageManagement Include="YesSql" Version="3.0.3" />
-    <PackageManagement Include="YesSql.Core" Version="3.0.3" />
-    <PackageManagement Include="YesSql.Abstractions" Version="3.0.3" />
-    <PackageManagement Include="YesSql.Filters.Abstractions" Version="3.0.3" />
-    <PackageManagement Include="YesSql.Filters.Query" Version="3.0.3" />
+    <PackageManagement Include="YesSql" Version="3.0.4" />
+    <PackageManagement Include="YesSql.Core" Version="3.0.4" />
+    <PackageManagement Include="YesSql.Abstractions" Version="3.0.4" />
+    <PackageManagement Include="YesSql.Filters.Abstractions" Version="3.04" />
```
Comment (agriffard) [inline src/OrchardCore.Build/Dependencies.props:51]: 3.04 => 3.0.4

**Comment by deanmarcussen** [comment]
Thanks for the typo fixes @agriffard ;)

> Filtering on Type = Menu (that does not have any results) and then on Culture = en returns an error:

If you have time today, I was never able to repro the above error. Think all the defaults should be sorted now.

**Comment by agriffard** [comment]
@deanmarcussen The error is still occurring.
Steps: OC instance with Saas recipe, create a Page content type, enable Content Localization, add a Culture (`en` in my case).

In the content items admin: Filter on type:Page => ok,  then filter on culture:en :
`SqliteException: SQLite Error 1: 'no such column: LocalizedContentItemIndex_a1.ModifiedUtc'.`

If I just filter on culture:en, it returns "No results found."

**Comment by agriffard** [comment]
When I filter on `status:Owner`, 2 versions of the same content items are displayed:
image

**Comment by deanmarcussen** [comment]
Fixed @agriffard

**Pull request merged**